src/components/Table/TableHead.js

```diff
@@ -25,42 +25,50 @@ import classNames from 'classnames'
 
 import { Button, Tip, Tooltip, TextTooltipTemplate } from 'igz-controls/components'
 
+import { SORT_PROPS } from 'igz-controls/types'
+
 const TableHead = React.forwardRef(
   ({ content, mainRowItemsCount, selectedItem, sortProps }, ref) => {
-    const { selectedColumnName, getSortingIcon, sortTable } = sortProps
-
-    const getHeaderClasses = (tableItemClass, headerId) =>
+    const getSortableHeaderCellClasses = (tableItemClass, headerId, index) =>
       classNames(
         'table-head__item',
         tableItemClass,
         'table__header-item-sortable',
-        selectedColumnName === headerId && 'table__header-item-sortable-active'
+        sortProps &&
+          sortProps.selectedColumnName &&
+          sortProps.selectedColumnName === headerId &&
+          'table__header-item-sortable-active',
```

Comment:
probably you can simplify this code
`sortProps?.selectedColumnName === headerId && 'table__header-item-sortable-active'`